addons/mail/static/src/models/follower/follower.js

```diff
@@ -77,16 +77,17 @@ function factory(dependencies) {
          * Remove this follower from its related thread.
          */
         async remove() {
-            const args = [[this.followedThread.id]];
+            const partner_ids = [];
+            const channel_ids = [];
             if (this.partner) {
-                args.push([this.partner.id]);
+                partner_ids.push(this.partner.id);
             } else {
-                args.push([this.channel.id]);
+                channel_ids.push(this.channel.id);
             }
             await this.async(() => this.env.services.rpc({
                 model: this.followedThread.model,
                 method: 'message_unsubscribe',
-                args
+                args: [[this.followedThread.id], partner_ids, channel_ids]
```

Comment:
Note as I see this: it would have been more robust to switch to kwargs while we were at it. But this is ok for now.